pkgs/tools/networking/webalizer/default.nix

```diff
@@ -2,24 +2,26 @@
 
 stdenv.mkDerivation rec {
   pname = "webalizer";
-  version = "2.23-05";
+  version = "2.23.08";
 
   src = fetchurl {
-    url = "ftp://ftp.mrunix.net/pub/webalizer/webalizer-${version}-src.tar.bz2";
-    sha256 = "0nl88y57a7gawfragj3viiigfkh5sgivfb4n0k89wzcjw278pj5g";
+    url = "http://ftp.debian.org/debian/pool/main/w/webalizer/webalizer_${version}.orig.tar.gz";
```

Comment:
Is there a reason to use the debian download instead of the previously used upstream ftp?

Reply:
Have you tried calling the previously used upstream ftp?

Reply:
Thanks for clarifying that. Maybe also add a note to the source about the original FTP being unreachable?

Reply:
```suggestion
    url = "https://ftp.debian.org/debian/pool/main/w/webalizer/webalizer_${version}.orig.tar.gz";
```